ksqldb-engine/src/main/java/io/confluent/ksql/query/QueryRegistryImpl.java

```diff
@@ -249,6 +249,7 @@ private void registerQuery(
       final QueryMetadata query,
       final boolean createAsQuery
   ) {
+    notifyCreate(serviceContext, metaStore, query);
```

Comment:
Just double checking here: do you think we would ever have any listeners that assumes the query is already in the registry map in its impl?

Reply:
no, they shouldn't need to either since we pass the query in.